roles/backpack_kube/tasks/main.yml

```diff
@@ -1,48 +1,75 @@
 ---
+- name: Set facts
+  set_fact:
+    operator_namespace: "{{ operator_namespace | default('backpack') }}"
```

Comment:
Curious why set facts here instead of using vars or group_vars ?

Reply:
It certainly could be set in group_vars or the like. That said i'd still probably have the set_fact for this here to set the default so it doesn't need to be defaulted in a dozen separate places.

Reply:
not sure i follow what you mean by you still have to set the fact to the default as you can still set the default in group_vars, the line can be copied and pasted into group_vars.

Reply:
That's fair. I was mostly trying to abstract the "default" away from where the user would normally go in to mess w/ so they couldn't delete the line and then go "why isn't it working." If you prefer I'm open  to moving it to group_vars

Reply:
IMHO group_vars allows the user to possibly overwrite (easily) vs statically assigning `backpack` as the operator_namespace.

Reply:
Updated